fiftyone/core/annotation.py

```
@@ -78,3 +81,107 @@ class AnnotationResults(BaseRunResults):
+    is_list = isinstance(field, fof.ListField)
+    if is_list:
+        field = field.field
+
+    if isinstance(field, fof.StringField):
+        try:
+            return {
+                "default": None,
+                "type": "checkbox" if is_list else "select",
+                "values": collection.distinct(field_name),
+            }
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)